new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java

```diff
@@ -197,11 +223,11 @@ public static V1Pod getPod(String namespace, String labelSelector, String podNam
             Boolean.FALSE // Watch for changes to the described resources.
         );
     for (V1Pod item : v1PodList.getItems()) {
-      if (item.getMetadata().getName().startsWith(podName.trim())) {
-        logger.info("Pod Name: " + item.getMetadata().getName());
-        logger.info("Pod Namespace: " + item.getMetadata().getNamespace());
-        logger.info("Pod UID: " + item.getMetadata().getUid());
-        logger.info("Pod Status: " + item.getStatus().getPhase());
+      if (item.getMetadata().getName().contains(podName.trim())) {
+        logger.info("Pod Name :" + item.getMetadata().getName());
```

Comment:
you were right the first time, space before colon, not after

Reply:
fixed